### ISSUE TITLE
Wired Switch in Wireless Mode

### ISSUE THREAD

**Issue by timdonovanuk** [Danielhiversen/PyXiaomiGateway#145]
Hi,

The wired Xiaomi wall switche (https://www.gearbest.com/alarm-systems/pp_610096.html?wid=1433363 + the others powdered models) support a mode referred to as "wireless mode" or sometimes "decoupled". 

In normal mode:

- physically pressing the switch disconnects the live lighting circuit. You can also control the switch remotely, turning the live lighting circuit on/off.

In wireless/decoupled mode:

- physically pressing the switch DOES NOT disconnect the live lighting circuit. It simply triggers an event on the gateway (configured via the Mi Home app). The live lighting circuit can still be controlled, but only via the app (or HA..).

This wireless/decoupled state (pushed) is not currently tracked in HA. It's been discussed in a few places and although some people mentioned the gateway doesn't actually send the wireless/decoupled state, it doesn't look like anyone ever actually logged an issue here, so I'm not sure if that's the case?

Cheers!

**Comment by syssi** [comment]
Correct. The button doesn't publish events in decoupled mode (at the LAN protocol **v1**!). As long as the firmware doesn't expose the events we are stuck.

**Comment by timdonovanuk** [comment]
Ah okay, just wanted to verify, thanks! That's a real bummer for my HA plans :(
